packages/hs-react-ui/src/enums/TextInputTypes.ts

```diff
@@ -0,0 +1,3 @@
+export enum TextInputTypes {
+    icon = 'icon'
+}
```

Comment:
looks like this isn't being used - let's delete the file

Reply:
resolved